lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -198,33 +226,59 @@ def get_udids(app_id)
 
       private
 
-      def v1_apps_url(app_id)
+      def v1alpha_apps_url(app_id)
         "/v1alpha/apps/#{app_id}"
       end
 
-      def release_notes_create_url(app_id, release_id)
-        "#{v1_apps_url(app_id)}/releases/#{release_id}/notes"
+      def v1_projects_url(project_number)
+        "/v1/projects/#{project_number}"
+      end
+
+      def v1_apps_url(project_number, app_id)
+        "#{v1_projects_url(project_number)}/apps/#{app_id}"
+      end
+
+      def aab_info_url(project_number, app_id)
+        "#{v1_apps_url(project_number, app_id)}/aabInfo"
+      end
+
+      def update_release_notes_url(project_number, app_id, release_id)
+        "#{v1_apps_url(project_number, app_id)}/releases/#{release_id}"
       end
 
-      def enable_access_url(app_id, release_id)
-        "#{v1_apps_url(app_id)}/releases/#{release_id}/enable_access"
+      def enable_access_url(project_number, app_id, release_id)
+        "#{v1_apps_url(project_number, app_id)}/releases/#{release_id}:distribute"
       end
 
-      def binary_upload_url(app_id)
-        "/app-binary-uploads?app_id=#{app_id}"
+      def binary_upload_url(project_number, app_id)
+        "/upload#{v1_apps_url(project_number, app_id)}/releases:upload"
       end
 
-      def upload_status_url(app_id, app_token)
-        "#{v1_apps_url(app_id)}/upload_status/#{app_token}"
+      def upload_status_url(operation_name)
+        "/v1/#{operation_name}"
       end
 
       def get_udids_url(app_id)
-        "#{v1_apps_url(app_id)}/testers:getTesterUdids"
+        "#{v1alpha_apps_url(app_id)}/testers:getTesterUdids"
```

Comment:
I didn't realize this was already supported in the old API -- to me this would dictate adding it to our build plan for GA.

Reply:
True, might be nice to pull it before releasing a new version if we wanted to rid this code of v1alpha references entirely (assuming we also get rid of `GetApp` in favor of app id parsing for project number).